libraries/common/effekt.effekt

```diff
@@ -590,6 +590,11 @@ extern pure def bitwiseShl(x: Int, y: Int): Int =
   chez "(ash ${x} ${y})"
   llvm "%z = shl %Int %x, %y ret %Int %z"
 
+extern pure def bitwiseShr(x: Int, y: Int): Int =
+  js "(${x} >> ${y})"
+  chez "(ash ${x} (- ${y}))"
+  llvm "%z = ashr %Int %x, %y ret %Int %z"
```

Comment:
Should the bitwise `shr` be arithmetic (`ashr`) or logical (`lshr`)? :)
(see https://llvm.org/docs/LangRef.html#bitwise-binary-operations)

Reply:
JS shifts are arithmetic, right? I think LLVM should use `ashr` for consistency.

Reply:
Yeah, you're right. I added a test suite just to make sure that they don't drift :)